### PR TITLE
WRO-8014: [POC] Enact WebXR Components - Code cleanup

### PR DESCRIPTION
**Pull request by vladut-clapou-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Code cleanup for Sandstone 3d components

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-8014

### CODE REVIEW

File: ImageItem3D/ImageItem3D.js [hunk 1]
```diff
@@ -7,10 +7,10 @@ import PropTypes from 'prop-types';
 import {useCallback, useImperativeHandle, useRef, useState} from 'react';
 import * as THREE from 'three';
 
-import Skinnable from '../Skinnable';
+import {get3DShape} from '../internal/util/util';
```
Comment (stanca-pop-lgp) [inline ImageItem3D/ImageItem3D.js:10]: You can shorten this to: import {get3DShape} from '../internal/util';
I would also recommend to name this "utils", it is more common.

File: ImageItem3D/ImageItem3D.js [hunk 2]
```diff
@@ -34,7 +34,7 @@ const ImageItem3DBase = kind({
 		position: [0, 0, 0],
 		selected: null,
 		setSelected: null,
-		setControlled: () => {}
+		setControlled: () => { }
```
Comment (stanca-pop-lgp) [inline ImageItem3D/ImageItem3D.js:37]: You can leave it without the empty space between curly brakets.

File: ImageItem3D/ImageItem3D.js [hunk 3]
```diff
@@ -87,11 +79,16 @@ const ImageItem3DBase = kind({
...
+				if (selected === index) {
+					setSelected(null);
+				} else {
+					setSelected(index);
+				}
 			} else {
-				setSelected(index);
+				return null;
 			}
+
```
Comment (stanca-pop-lgp) [inline ImageItem3D/ImageItem3D.js:91]: You can remove this empty line. (it's the same structure as for 'handlePointerDown' for example, and we don't have an empty line there - line 69)

File: ImageItem3D/ImageItem3D.js [hunk 4]
```diff
@@ -198,10 +195,5 @@ const ImageItem3DBase = kind({
 	}
 });
 
-const ImageItem3D = Skinnable({prop: 'skin'}, ImageItem3DBase);
```
Comment (stanca-pop-lgp) [inline ImageItem3D/ImageItem3D.js:198]: Please remove one empty line.

File: samples/sampler/stories/default/Button3D.js [hunk 5]
```diff
@@ -1,15 +1,14 @@
-import Button3D, {Button3DBase} from '@enact/sandstone/Button3D';
+import  Button3D from '@enact/sandstone/Button3D';
```
Comment (stanca-pop-lgp) [inline samples/sampler/stories/default/Button3D.js:1]: Please remove the empty space before Button3D.

**Review by stanca-pop-lgp — CHANGES_REQUESTED** [review]
I left some comments. Thanks!

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

File: Button3D/Button3D.js [hunk 6]
```diff
@@ -5,8 +5,7 @@ import {useCallback, useEffect, useRef, useState} from 'react';
 import * as THREE from 'three';
 
 import iconList from '../Icon/IconList.js';
-import Skinnable from '../Skinnable';
-
+import {get3DShape} from '../internal/util';
 import sandstoneIcons from '../fonts/Sandstone_Icons.woff';
```
Comment (alexandrumorariu) [inline Button3D/Button3D.js:9]: Order imports here

**Review by alexandrumorariu — APPROVED** [review]
LGTM
